### ISSUE TITLE
Suggestion for finetune with unsloth

### ISSUE THREAD

**Issue by ILG2021** [canopyai/Orpheus-TTS#175]
Hello, has anyone finetuned Orpheus_tts?
I have a 15.5 hours lao dataset, single speaker, how many epoches should I train that I can get a good result? and how about the batch size? The colab notebook unsloth gives has a batch size 1 and gradient_accumulation_steps 4, is it right for normal train?

**Comment by ILG2021** [comment]
> You can do full 8bit finetuning using unsloth on 24GB with batch size 2 - I am using gradient_accumulation_steps 4 - so effective batch_size=8. It takes around 23,8 GB...
> 
> Everything works fine so far...

Ok thank you. But unsloth only gives a 4 bit pretrain Orpheus-TTS model, and if I use batch_size = 2, I need to use a batch data_collator. how many epochs should I train to get a good result?

**Comment by ILG2021** [comment]
> > if I use batch_size = 2, I need to use a batch data_collator
> 
> yes, you will need to use it
> 
> ```
> data_collator = DataCollatorForSeq2Seq(
>     tokenizer,
>     model=model,
>     padding="longest",
>     pad_to_multiple_of=8,
>     return_tensors="pt"
> )
> ```
> 
> > But unsloth only gives a 4 bit pretrain Orpheus-TTS model
> 
> you can download the original model from canopy repo and use load_in_8bit = True and full_finetuning = True paramters in unsloth
> 
> I am training for 3 epoch using cosine lr scheduler, LEARNING_RATE = 5e-5

Thank you very much.
